### PR TITLE
Added banner component

### PR DESCRIPTION
**Pull request by harshita-jaiswal**
Changes Made

- Added banner in the header component
- Added banner.jpeg in assets 

Feedback

Does the UI look fine?  I've removed the gaps in the grid, but we can always adjust it further if needed.

### CODE REVIEW

File: NBA-Stats-Dashboard/src/components/Header.jsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 function Header() {
   return (
     <div className='majorSection header'>
-      <h1>The Bannner will go here. Which is inside the Header Component.</h1>
+      <img
+          src="../src/assets/banner.jpeg"
+          height="200px"
+          width="100%"
```
Comment (JoshuaWachana) [inline NBA-Stats-Dashboard/src/components/Header.jsx]: I feel like a 100% width makes the banner image look distorted at certain screen sizes, especially large (desktop) screen sizes.

**Review by JoshuaWachana — COMMENTED** [review]
Hello Harshita, thanks for your work so far!

I like the image chosen for the banner image, but the way it is currently implemented makes it look a bit distorted at certain screen sizes. Perhaps we should use a fixed width and height  for the image that change based upon the screen size. We should consider using media queries for this. Also the banner image does not have to be the full width of the page, banner, or header. If there is some left over space within the header component we can just set a background color similar to that of the image. I think there may be better options than a distorted looking image.

**Comment by JoshuaWachana** [comment]
Also, you may consider using a svg over a png image, as they are sometimes easier to work with when it comes to resizing and keeping the image quality intact. If so, consider: [https://seeklogo.com/vector-logo/327223/nba](https://seeklogo.com/vector-logo/327223/nba)

File: NBA-Stats-Dashboard/src/components/Header.jsx (hunk 1 above)
Comment (harshita-jaiswal) [reply NBA-Stats-Dashboard/src/components/Header.jsx]: It works fine with the 1300px screen size. For very large screens we can consider using better image or SVG images in improvement phase. It's good to go for the first phase.

**Comment by harshita-jaiswal** [comment]
> Also, you may consider using a svg over a png image, as they are sometimes easier to work with when it comes to resizing and keeping the image quality intact. If so, consider: https://seeklogo.com/vector-logo/327223/nba

Noted

**Comment by JoshuaWachana** [comment]
Hello Harshit, thanks for making the changes. I think it is good enough for now, we can make improvements later on if we have the time.

**Review by JoshuaWachana — APPROVED** [review]

**Pull request merged**